fendermint/app/src/app.rs

```diff
@@ -446,11 +446,11 @@ where
             anyhow!("failed to load custom actor bundle CAR from {custom_actors_bundle:?}: {e}")
         })?;
 
-        let state = FvmGenesisState::new(
+        let state = FvmGenesisState::new_state(
```

Comment:
Is this rename necessary? "state::new" says the same thing as "state::new_state" already.

Reply:
because `new` is taken and I dont want to update this method that will break all the tests. If this approach is ok, I will replace `new`.